src/main/java/me/hydos/lint/item/LintItems.java

```diff
@@ -72,13 +80,24 @@
 	public static final Item DUSKBLADE = new Item(new Item.Settings().group(ItemGroups.TOOLS).maxCount(1).rarity(Rarity.EPIC));
 	public static final Item ATTUNER = new Item(new Item.Settings().group(ItemGroups.TOOLS).maxCount(1));
 	public static final Item QUESTBOOK = new Item(new Item.Settings().group(ItemGroups.ITEMS).maxCount(1).rarity(Rarity.RARE));
+	public static final Item GUIDE_BOOK = new Item(new Item.Settings().group(ItemGroups.ITEMS).maxCount(1).rarity(Rarity.UNCOMMON).fireproof()) {
```

Comment:
could you make this an class instead of an anonymous class?

Reply:
but why 😢

Reply:
because it looks nicer in code

Reply:
will do